scripts/test_screeps_tencent_batch_rl_runner.py

```
@@ -898,6 +922,130 @@ def test_verified_remote_policy_update_accepts_positive_update_with_collected_ar
+        cases.append(
+            (
+                "next candidate bool",
+                next_bool_policy_update,
+                "policyUpdate.nextCandidatePolicy.parameters values must be finite numbers",
+            )
+        )
+
+        with tempfile.TemporaryDirectory() as temp_dir:
+            root = Path(temp_dir)
+            for label, policy_update, expected_error in cases:
+                with self.subTest(label=label), self.assertRaisesRegex(
```

Ruff (0.15.13)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)